react/tic-tac-toe/src/app/screens/Game/components/Matches/index.js

```diff
@@ -1,43 +1,60 @@
+/* eslint-disable no-extra-parens */
 import React, { Component, Fragment } from 'react';
+import { connect } from 'react-redux';
+import { func, bool, arrayOf, shape } from 'prop-types';
 import Spinner from 'react-spinkit';
 
 import MatchesService from '../../../../../services/MatchesService';
+import actionsCreators from '../../../../../redux/matches/actions';
 
 import styles from './styles.module.scss';
 import { PLAYER_ONE, PLAYER_TWO } from './constants';
 
 const getWinnerClass = isWinner => (isWinner ? styles.winner : '');
 
 class MatchesList extends Component {
-  state = {
-    val: [],
-    loading: true
-  };
-
   componentDidMount() {
     const match = MatchesService.getMatches();
```

Comment:
You are calling the getmatches service twice. You should call it once, in the action.